dev/dags/example_jinja2_template_dag.py

```diff
@@ -0,0 +1,16 @@
+import os
+from pathlib import Path
+
+# The following import is here so Airflow parses this file
+# from airflow import DAG
+import dagfactory
+
+DEFAULT_CONFIG_ROOT_DIR = "/usr/local/airflow/dags/"
```

Comment:
This path will no longer work on Astro with Astro Executor in Airflow 3: https://github.com/astronomer/astronomer-cosmos/issues/1829

Reply:
Right. But this default value can be overridden using an environment variable when needed for specific environments. If you want me to change default value please let me know